src/modules/45-dashboards/pages/DashboardsContext.tsx

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2022 Harness Inc. All rights reserved.
+ * Use of this source code is governed by the PolyForm Shield 1.0.0 license
+ * that can be found in the licenses directory at the root of this repository, also available at
+ * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
+ */
+
+import React, { useState, useContext } from 'react'
+import { useParams } from 'react-router-dom'
+import { useStrings } from 'framework/strings'
+import routes from '@common/RouteDefinitions'
+import type { AccountPathProps } from '@common/interfaces/RouteInterfaces'
+
+interface DashboardsContextProps {
+  breadcrumbs: { url: string; label: string }[]
+  includeBreadcrumbs: (breadcrumbs: { url: string; label: string }[]) => void
+}
+
+const DashboardsContext = React.createContext<DashboardsContextProps>({} as DashboardsContextProps)
+
+export function DashboardsContextProvider(props: React.PropsWithChildren<unknown>): React.ReactElement {
+  const { getString } = useStrings()
+  const { accountId } = useParams<AccountPathProps>()
+
+  const standardBreadcrumbs = [
```

Comment:
Super minor, but might be worth typing this array
```suggestion
  const standardBreadcrumbs: Breadcrumb[] = [
```